### ISSUE TITLE
can't get to setup page on auto installer on 20.04

### ISSUE THREAD

**Issue by bjquinn** [automatic-ripping-machine/automatic-ripping-machine#587]
- [X] I have checked the wiki
- [X] I have searched for similar issues

**Describe the bug**
First-time setup never launches when running automatic setup.

**To Reproduce**
1. Follow instructions for automatic setup for Ubuntu 20.04
2. First time setup never launches
3. Manually loading http://[ip]:8080/setup in the browser results in "Unable to connect Firefox can’t establish a connection to the server"

### Environment
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.5 LTS"

arm@fireflash:/opt/arm$ git branch
* (no branch)
arm@fireflash:/opt/arm$ git log -1
commit 66b784e95437ab38e81425ce06f86f7e9afcc578 (HEAD, tag: v2.6.7, origin/v2_devel)
Author: wolfy <wolfy@shitwolfymakes.com>
Date:   Mon Sep 26 23:58:18 2022 -0600

    dev branch now properly checked out during clone (#580)

There doesn't appear to be any errors in the console when installing.  A few warnings like the following:

WARNING: The script apprise is installed in '/home/arm/.local/bin' which is not on PATH.

**Comment by bjquinn** [comment]
Oh ok awesome, thx.  So all I need to do is insert a disc and it should get started?

**Comment by shitwolfymakes** [comment]
You might want to fiddle with the settings to get them to your liking, but otherwise yeah.

Per the wiki, the default username and password are both `arm`

**Comment by bjquinn** [comment]
Ok thanks.  I went ahead and put in a disc and out a few times to see what would happen and I couldn't see anything in empty.log.

But I saw that sudo systemctl status armui was saying the following:

```
Oct 06 21:02:55 fireflash python3[107671]: FileNotFoundError: [Errno 2] No such file or directory: '/etc/arm/config/abcde.conf'
Oct 06 21:02:55 fireflash systemd[1]: armui.service: Main process exited, code=exited, status=1/FAILURE
Oct 06 21:02:55 fireflash systemd[1]: armui.service: Failed with result 'exit-code'.
```

So I did the following, and that seemed to fix it:

`ln -s /etc/.abcde.conf /etc/arm/config/abcde.conf`

Also, I didn't seem to notice in the wiki (perhaps I missed it, though) how to log into the Flask UI, but was able to guess that it was at http://[myip]:8080.  Specifically NOT http://localhost:8080, that did not work, even though I'm on the same machine ARM is running on.

And the default un/pw seemed to be admin/password.  arm/arm didn't work.

I also noticed that 51-automedia.rules was 755 perms, and the other files in the same directory were 644, so I changed that and reloaded udev, but that didn't seem to make any difference, so I don't think that was it.

One last question.  As I start digging around conf files for preferences, are the only ones to look at arm.yaml and abcde.conf?  And I suppose putting the makemkv key in place and apprise.yaml if I decided to do that.  Is there any other conf files I should be looking at?

Also worth noting the wiki says it's at /opt/arm/arm.yaml, but it seemed to be at /etc/arm/config/arm.yaml.  I had a broken link at /etc/arm/arm.yaml to /opt/arm/arm.yaml, which wasn't there.  I did do the manual install first before the auto install, so maybe there's some overlapping stuff here.

Thanks!

**Comment by shitwolfymakes** [comment]
Closed by #599

**Comment by puujam** [comment]
I encountered this same issue today and the directions for fixing it by creating a symbolic link above didn't work for me. On my system 
`/etc/.abcde.conf`
did not exist but
`/etc/abcde.conf` (no . at start of file name)
did. Changing the link to the file which did exist allowed armui to run.
